lua/telescope/actions/init.lua

```diff
@@ -671,7 +671,7 @@ local entry_to_qf = function(entry)
 
   return {
     bufnr = entry.bufnr,
-    filename = from_entry.path(entry, false),
+    filename = entry.path,
```

Comment:
`from_entry.path()` would fall back to `entry.filename` and `entry.value` in that order, but I don't believe the quickfix/loclist actions need that as well.

Reply:
I can reproduce that `vim.fn.fnameescape` does not work well with the quickfix list (eg as mentioned space in the fname or something like `test%.lua`). 

However, this breaks sending outputs from other pickers to the quickfix list, e.g. `lsp_{document, workspace}_{diagnostics, symbols}` as `from_entry` resolves the filename attribute.

Maybe we should add an option to not escape the path to `from_entry` instead? Not sure.